examples/hoa-renderer.html

```diff
@@ -0,0 +1,166 @@
+    the appropriate SH-MaxRe HRIRs.</p>
+  <div id="secSource">
+    <p>NOTE: Use headphones for the full-sphere surround sound.</p>
+    <h2>Orient the head by selecting the horizontal and vertical controls.</h2>
+    Azimuth: <span id="eLabelAzimuth">0</span><br/>
+    <input oninput="rotateScene();" onchange="rotateScene();"
+      id="eRangeAzimuth" type=range min="-180" max="180" value="0" step="1"/>
+    <br/>
+    Elevation: <span id="eLabelElevation">0</span><br/>
+    <input oninput="rotateScene();" onchange="rotateScene();"
+      id="eRangeElevation" type=range min="-90" max="90" value="0" step="1"/>
+    <br/>
+    Gain (dB): <span id="eLabelGain">0</span><br/>
+    <input oninput="adjustGain();" onchange="adjustGain();"
+      id="eRangeGain" type="range" min="-60" max="20" value="0" step="1"/>
+    <br/>
+    <button id="eButtonPlayback" disabled>Play</button>
+  </div>
+
+  <script>
+    var hoaRenderer;
+    var audioSources = [
+      {name : 'ch0-ch7', url : 'resources/3oa_example_ch0-ch7.wav'},
+      {name : 'ch8-ch15', url : 'resources/3oa_example_ch8-ch15.wav'}];
+    var bufferSource;
+    var startTime = 0;
+    var offsetTime = 0;
+    var buttonPlayback;
+    var bufferGain;
+
+    function crossProduct(a, b) {
+      c = [];
+      c[0] = a[1] * b[2] - a[2] * b[1];
+      c[1] = a[2] * b[0] - a[0] * b[2];
+      c[2] = a[0] * b[1] - a[1] * b[0];
+      return c;
+    }
+
+    function normalize(a) {
+      var n = Math.sqrt(a[0] * a[0] + a[1] * a[1] + a[2] * a[2]);
+      a[0] /= n;
+      a[1] /= n;
+      a[2] /= n;
+      return a;
+    }
+
+    function adjustGain() {
+      var gainDB = document.getElementById("eRangeGain").value;
+      document.getElementById('eLabelGain').textContent = gainDB;
+      var gainLin = Math.pow(10, gainDB / 20);
+      bufferGain.gain.value = gainLin;
+    }
+
+    function rotateScene() {
+      var azi = document.getElementById('eRangeAzimuth').value;
+      var ele = document.getElementById('eRangeElevation').value;
+      document.getElementById('eLabelAzimuth').textContent = azi;
+      document.getElementById('eLabelElevation').textContent = ele;
+
+      // Standard View Matrix
+      var matrix = new Array(9);
+      var beta = -azi / 180 * Math.PI;
+      var alpha = -ele / 180 * Math.PI;
+
+      matrix[0] = Math.cos(alpha) * Math.cos(beta);
+      matrix[1] = Math.sin(alpha) * Math.cos(beta);
+      matrix[2] = -Math.sin(beta);
+
+      matrix[3] = -Math.cos(alpha);
+      matrix[4] = Math.cos(alpha);
+      matrix[5] = 0;
+
+      matrix[6] = Math.cos(alpha) * Math.sin(beta);
+      matrix[7] = Math.sin(alpha) * Math.sin(beta);
+      matrix[8] = Math.cos(beta);
+
+      hoaRenderer.setRotationMatrix(matrix);
+    }
+
+    var onLoad = function () {
+      var audioContext = new AudioContext();
+      var buffer;
+      bufferGain = audioContext.createGain();
+      Omnitone.loadAudioBuffers(audioContext, audioSources)
+        .then(function (result) {
```

Comment:
Replace the anonymous function with the arrow function: `function (result) {` to `(result) => {`.